### ISSUE TITLE
Add installation instructions to the docs

### ISSUE THREAD

**Issue by danielo515** [gcanti/fp-ts#1912]
## 📖 Documentation
I was very surprised to see that there are no installation instructions in the "real" documentation of the project. It may sound silly, but there are many ways to install packages nowadays, so this information is still relevant. For example, I could think about this all possible installation methods:
- `npm i fp-ts`
- `npm i @fp-ts/Option @fp-ts/core`
Etc.
I suggest to add them quite early in the main page: https://gcanti.github.io/fp-ts/

**Comment by akbatra567** [comment]
@danielo515, Agreed. Seems like a great idea to add the installation instruction. 
I feel there is also no page like getting started, which I've seen in a lot of libraries. There it generally consists of installation instructions, pre-requisites and other details.

**Comment by aymenhmaidiwastaken** [comment]
Hi\! I'd like to work on this issue. I'll add clear installation instructions to the docs covering npm/yarn/pnpm install commands, the required TypeScript configuration, and a minimal working example. Should be a straightforward addition — I'll open a PR soon.

**Comment by aymenhmaidiwastaken** [comment]
I'd like to work on this. I'll add a Getting Started guide to the docs covering installation, TypeScript config requirements, and a minimal example to help new users get up and running.
